build/Targets/Versions.props

```diff
@@ -22,9 +24,10 @@
     <BuildNumber Condition="'$(BuildNumber)' == ''">$(BUILD_BUILDNUMBER)</BuildNumber>
     <Error Condition="'$(BuildNumber)' == '' AND '$(OfficialBuild)' == 'true'">A build number must be specified for a signed build.</Error>
 
-    <!-- When a build number is not specified, then we should default back to '00000000.0', which is a build number in the
-         same format as provided by MicroBuild v2, but with all digits set to zero. -->
-    <BuildNumber Condition="'$(BuildNumber)' == ''">00000000.0</BuildNumber>
+    <!-- When a build number is not specified, then we should default back to '00065535.0', which is a build number in the
+         same format as provided by MicroBuild v2, but set to a number in which MicroBuild v2 will produce and that is
+         still compattible with the Win32 file version limitations. -->
+    <BuildNumber Condition="'$(BuildNumber)' == ''">00065535.0</BuildNumber>
```

Comment:
I would rather this say "but greater than any that MicroBuild v2 will produce, and is still compatible with the Win32 file version limitations."